centreon/tests/api/features/UserFilter.feature

```diff
@@ -10,6 +10,9 @@ Feature:
   Scenario: User filters
     Given I am logged in
 
+    When I send a GET request to '/api/v21.10/users/filters/unknown-page'
```

Comment:
we should write postman tests instead of behat as we are in the process of migrating them